src/dev/ci_setup/setup.sh

```diff
@@ -46,14 +59,25 @@ else
 
   echo " -- downloading node.js from $nodeUrl"
   mkdir -p "$nodeDir"
-  curl --silent "$nodeUrl" | tar -xz -C "$nodeDir" --strip-components=1
+  if [[ $OS == 'win' ]]; then
+    nodePkg="$nodeDir/${nodeUrl##*/}"
+    curl --silent -o $nodePkg $nodeUrl
+    unzip -jqo $nodePkg -d $nodeDir
+  else
+    curl --silent "$nodeUrl" | tar -xz -C "$nodeDir" --strip-components=1
+  fi
+
 fi
 
 
 ###
 ### "install" node into this shell
 ###
-export PATH="$nodeDir/bin:$PATH"
+if [[ $OS == 'win' ]]; then
+  export PATH="$PATH:$nodeBin"
+else
+  export PATH="$nodeBin:$PATH"
```

Comment:
Why do we prepend, vs append on Linux?

Reply:
I was getting node path errors if the node bin dir it was not listed at the end of the windows path

Reply:
I am not seeing any errors with setting it first - if anything I would expect an issue if there was a node executable already in your path. Setting it first would give it precedence. What is the exact error you're seeing?

Reply:
I am not sure why :)   It was this type of error: 
Error: 'module' is undefined
Code: 800A1391
Source: Microsoft JScript runtime error

It is possible I had another node version installed when the problem occurred, I can't recall right now.  I will remove it for now and if the problem comes up again we can fix it at that time.